libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -879,7 +879,8 @@ ASTPointer<StructuredDocumentation> ASTJsonImporter::createDocumentation(Json::V
 
 Json::Value ASTJsonImporter::member(Json::Value const& _node, string const& _name)
 {
-	astAssert(_node.isMember(_name), "Node '" + _node["nodeType"].asString() + "' (id " + _node["id"].asString() + ") is missing field '" + _name + "'.");
+	if (!_node.isMember(_name))
```

Comment:
I don't think we should remove this and the other asserts

Reply:
@Marenz If we remove members with `null` as value, from JSON output, then this assert will have to be removed as well. Otherwise all these removed members will show up as "is missing field 'member'" errors

Reply:
Ah, it's the importer. Fair enough. However, we should make sure that we only allow missing members at places where we removed it previously